community/sigs/02_sig_infra.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "SIG Infra"
+---
+
+__Focus:__ Creating automation for various Eclipse Tractus-X topics like CI or release. 
+
+__Repository:__ [eclipse-tractusx/sig-infra](https://github.com/eclipse-tractusx/sig-infra)
+
+## Intro
+
+SIG infra is striving to improve the developer experience in Eclipse Tractus-X, by working on different automation topics.
+We are also providing guidance on how to work with [Helm](https://helm.sh/) and support the [e2e-testing](https://github.com/eclipse-tractusx/e2e-testing)
+approach, that will be based on a umbrella Chart and Helm tests.
+
+## Dash automation
+
+A topic, every Tractus-X repo has to take care of, is legal compliance.
+One of the aspects is an up-to-date `DEPENDENCIES` file, generated by [eclipse dash-license](https://github.com/eclipse/dash-licenses).
```

Comment:
```suggestion
One of the aspects is an up-to-date `DEPENDENCIES` file, generated by [eclipse dash-license](https://github.com/eclipse/dash-licenses) tool.
```

Reply:
I see, I missed the 's' at the end of dash-licenses. Would it be fine to add that and omit "tool". 
I think "generated by <product-name>" is enough

Reply:
yeah, that should be ok, too